dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -100,3 +100,71 @@ export const makeWindowGuardian = (
 		GAData: dcrDocumentData.GA,
 	};
 };
+
+const makeFrontWindowGuardianConfig = (
+	Front: FrontType,
+): WindowGuardianConfig => {
```

Comment:
I think we typically keep variables lowercase?

```suggestion
const makeFrontWindowGuardianConfig = (
	front: FrontType,
): WindowGuardianConfig => {
```

But we could also just de-structure the bits we’re using: 

```suggestion
const makeFrontWindowGuardianConfig = ({
	config,
	editionId,
}: FrontType): WindowGuardianConfig => {
```